docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+```
+Put the kubeconfig(s) on every node.  The examples later in this
+guide assume kubeconfigs in `/var/lib/kube-proxy/kubeconfig` and
+`/var/lib/kubeletproxy/kubeconfig`.
+
+#### Selecting Binaries
+You will need binaries for:
+  - etcd
+  - docker or rkt
+  - kubelet
+  - kube-proxy
+  - kube-apiserver
+  - kube-controller-manager
+  - kube-scheduler
+
+You can use a binary release (recommended) or build your own following the instructions in the
+[Developer Documentation]( ../devel/README.md).  Only using a binary release is covered in this guide.
+
+Download the [latest binary release](
+https://github.com/GoogleCloudPlatform/kubernetes/releases/latest) and unzip it.
+Then locate `./kubernetes/server/kubernetes-server-linux-amd64.targz` and unzip *that*.
+Then, within the second set of unzipped files, locate `./kubernetes/server/bin`, which contains
+all the necessary binaries.
+
+The kube-apiserver, kube-controller-manager, and kube-scheduler, you will run as pods,
+so there needs to be a docker image in whatever docker repository you are using.
+Options:
+  - Use images hosted on Google Container Registry (GCR):
+       - e.g `gcr.io/google_containers/kube-apiserver:$TAG`, where `TAG` is the latest
+          release tag, which can be found on the [latest releases page](
+          https://github.com/GoogleCloudPlatform/kubernetes/releases/latest).
+    $TAG is the same tag as the release tag you are using for kubelet and kube-proxy.
+  - Build and push your own images.
+    - Necessary if you are using a private registry.
+    - The release contains files such as `./kubernetes/server/bin/kube-apiserver.tar` which
+      can be converted into docker images using a command like
+      `tar -C kube-apiserver -c . | docker import - kube-apiserver`
+    - *TODO*: test above command.
+
+You may also run etcd in a pod, in which case you can:
+  - Use a GCR image, such as `gcr.io/google_containers/etcd:2.0.12`
+  - Build your own image.
+  - Build your own image using `kubernetes/cluster/images/etcd/Dockerfile` and
+   `kubernetes/cluster/images/etcd/Makefile`
+
+The remainder of the document assumes that the image identifiers have been chosen and stored in corresponding env vars.  Examples (replace with latest tags and adjust if using private registry):
+  - `APISERVER_IMAGE=gcr.io/google_containers/kube-apiserver:$TAG`
+  - `SCHEDULER_IMAGE=gcr.io/google_containers/kube-scheduler:$TAG`
+  - `CNTRLMNGR_IMAGE=gcr.io/google_containers/kube-controller-manager:$TAG`
+
+### Configuring Base Node Software
+This section discusses how to configure machines to be kuberetes nodes. 
+
+You should run three daemons on every node:
+  - docker or rkt
+  - kubelet
+  - kube-proxy
+
+You will also need to do assorted other configuration on top of a
+base OS install.
+
+One possible starting point is to setup a cluster using an existing Getting
+Started Guide and then copy the init.d scripts or systemd unit files from that
```

Comment:
Revise to:
...Started Guide. After getting a cluster running, you can then copy the......